demo/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html>
 <!--
```

Comment:
Admittedly stylistic since it's valid either way but caught my vim editor off-guard (opinionated or outdated check by original author  being the traditional way it's declared before HTML5). I'll revert this as well.